src/ui/components/connect/listConnections/listConnections.html

```diff
@@ -4,7 +4,7 @@
   </div>
   <ul class="connection-list">
     <li ng-repeat="connection in connections" ng-class="{ 'active' : connection.selected }" ng-show="connections.length">
-      <a class="name" href="#" ng-click="selectConnection(connection, $event)">
+      <a class="name" href="#" ng-click="selectConnection(connection, $event)" ng-dblclick="selectConnection(connection, $event);connect($event)">
```

Comment:
ooops I totally missed which files you added this too, I thought this was for the sidebar :/ ignore my previous comment

but yea I agree with your comment about adding a blue highlight around the current select connection.